### DISCUSSION TITLE
Great plugin, but i got some problem!

### DISCUSSION THREAD

**Discussion by hoangpbc** [mattieserver/netbox-topology-views#325] [General]
I install netbox_topology_views v3.5 on my netbox 3.5.2
Everythings work fine but https://my-netbox/api/schema/swagger-ui/ is not.
I got Fetch error
Internal Server Error /api/schema/.
 I try to disable plugin then https://my-netbox/api/schema/swagger-ui/ working fine.
 please help?

**Comment by mattieserver** [comment]
See https://github.com/mattieserver/netbox-topology-views/issues/295
